### PR TITLE
DIV-3889 AOS Back button bug

### PR DESCRIPTION
**Pull request by ravinderpal**
# Description

Removed smart redirect to go from initial page to review application page

https://tools.hmcts.net/jira/browse/DIV-3889

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: steps/respond/Respond.step.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Respond extends Interstitial {
   }
 
   next() {
-    return goTo(this.journey.steps.ReviewApplication);
+    return redirectTo(this.journey.steps.ReviewApplication);
```
Comment (hosainnet) [inline steps/respond/Respond.step.js]: Just wondering if it's possible to test that we're using the right one?

**Comment by tchow8** [comment]
I think that your ticket isn't just for the review-application page but for every page? Maybe can confirm with BA, I thought that the intended action is meant to always go to the next question regardless of what page you are on.

File: steps/respond/Respond.step.js (hunk 1 above)
Comment (ravinderpal) [reply steps/respond/Respond.step.js]: On manual testing it works well. In addition, DN is using redirectTo() in every step to go to the next page

**Comment by ravinderpal** [comment]
> I think that your ticket isn't just for the review-application page but for every page? Maybe can confirm with BA, I thought that the intended action is meant to always go to the next question regardless of what page you are on.

@tchow8 Had a discussion regarding that with business (please refer to my last comment on the ticket) and at this stage they are happy to keep the smart redirect

**Comment by hosainnet** [comment]
As per discussion in DIV-3889 and IRL, it's not clear whether this should be page, never or across the whole app. Closing for now until requirement is clear

**Pull request closed without merging**